### DISCUSSION TITLE
show year in album lists

### DISCUSSION THREAD

**Discussion by sentriz** [tamland/airsonic-refix#122] [Ideas]
i find it nice to see what year an album was released in an AlbumList. what do you think?
i'm not certain about the style yet, but is there any interest in such a feature

from my personal fork i have this:
![Screenshot 2023-09-22 at 19-53-22 Airsonic (refix)](https://github.com/tamland/airsonic-refix/assets/6832539/806cbdb8-590e-4fcf-89b0-a0e61564d462)

**Comment by fletchowns** [comment]
I'd love this! The ability to sort the albums of a particular artist by year would be awesome too - I often find it's my preferred way of diving into an artists material, start with their earliest works.
